griptape/drivers/image_generation_model/base_diffusion_pipeline_image_generation_model_driver.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+from typing import TYPE_CHECKING, Any, Optional
+
+from attrs import define
+
+if TYPE_CHECKING:
+    from PIL.Image import Image
+
+
+@define
+class BaseDiffusionPipelineImageGenerationModelDriver(ABC):
```

Comment:
Do we need a new base class? Can we instead use `BaseImageGenerationModelDriver`?

Reply:
The interface difference between these two model driver types is substantial. Drivers for remote models package inputs and unpackage outputs, drivers for remote models package inputs and prepare diffusion pipelines. We could certainly merge these together, but I think that would increase the overall awkwardness of this implementation.

Reply:
Understood. What do you think about renaming to `BaseDiffusersImageGenerationModelDriver` since it's very specific to the library? Or `BaseHuggingfacePipelineImageGenerationModelDriver`? We should aim for consistency with `HuggingfacePipelinePromptDriver` since that uses the sister library `tranformers`.

Reply:
Understood. Should any of these model driver classes be renamed with the new name for `HuggingFacePipelineImageGenerationDriver`?